rebar.lock

```diff
@@ -32,5 +40,20 @@
  {<<"poolboy">>, <<"392B007A1693A64540CEAD79830443ABF5762F5D30CF50BC95CB2C1AAAFA006B">>},
  {<<"ranch">>, <<"6B1FAB51B49196860B733A49C07604465A47BDB78AA10C1C16A3D199F7F8C881">>},
  {<<"riak_client">>, <<"6B78373E576D80C7020BCBDA3B2C102221D32863580404F59D8791E5CDC19E79">>},
- {<<"riak_pb">>, <<"48FFBF66DBB3F136AB9A7134BAC4E496754BAA5EF58C4F50A61326736D996390">>}]}
+ {<<"riak_pb">>, <<"48FFBF66DBB3F136AB9A7134BAC4E496754BAA5EF58C4F50A61326736D996390">>}]},
+{pkg_hash_ext,[
```

Comment:
Was this entry added automatically by rebar3? What is this entry for? It mentions other versions of already existing dependencies (e.g. riak_pb). Why?

Reply:
Yes, all modifications to this file were automatically added by rebar3. I think it's because I'm using a more recent version of rebar3, or at least that's what some warnings in the travis build log suggest. Should I use a specific version of rebar3?